python_modules/dagster/dagster/_grpc/types.py

```diff
@@ -405,18 +405,27 @@ class PartitionArgs(
         "_PartitionArgs",
         [
             ("repository_origin", RemoteRepositoryOrigin),
+            ("job_name", str),
+            # This is here for backcompat. it's expected to always be f"{job_name}_partition_set".
             ("partition_set_name", str),
             ("partition_name", str),
             ("instance_ref", Optional[InstanceRef]),
+            # This is introduced in the same release that we're making it possible for an asset job
```

Comment:
@gibsondan I added some additional explanation here. Do you think this makes it clear? Open to further suggestions.

Reply:
Yeah this works for me!